File: swedish-cc/2025/kth-challenge/cheaterdetection/cheaterdetection.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  cin.tie(0)->sync_with_stdio(0);

  int n;
  cin >> n;
  while (n--) {
    double x;
    cin >> x;

    int cnt = floor(x / 0.015);
    if (floor(100 * 0.015 * cnt) / 100 == x ||
        floor(100 * 0.015 * (cnt + 1)) / 100 == x) {
      cout << "VALID" << '\n';
    } else {
      cout << "IMPOSSIBLE" << '\n';
    }
  }
}

File: swedish-cc/2025/kth-challenge/cheaterdetection/cheaterdetection.py
from math import floor

n = int(input())

for _ in range(n):
    num = float(input())

    cnt = round(num / 0.015)

    if (
        floor(100 * cnt * 0.015) / 100 == num
        or floor(100 * (cnt - 1) * 0.015) / 100 == num
        or floor(100 * (cnt + 1) * 0.015) / 100 == num
    ):
        print("VALID")
    else:
        print("IMPOSSIBLE")
